packages/daemon/src/vault/Vault.js

```diff
@@ -172,9 +172,9 @@ export default class Vault {
       apps: new AppsRepository(),
       identities: new IdentitiesRepository(),
       settings: {
-        bzzURL: 'http://swarm-gateways.net',
-        pssURL: 'ws://localhost:8546',
-        ethURL: INFURA_URLS.ropsten,
+        bzzURL: 'http://mainframe-gateways.net:8500',
+        pssURL: 'ws://mainframe-gateways.net:8546',
+        ethURL: INFURA_URLS.mainnet,
```

Comment:
@PaulLeCam What does this url apply to? Isn't there a selector in the launcher UI that the user controls? I assumed this was going all the way through the daemon